tests/tools/oesign-engine/test-enclave/oesign-custmize-config.py

```diff
@@ -32,22 +32,64 @@ def generate_neg_config():
     fh.writelines(cfg_seq)
     fh.close()
 
+def generate_invalid_debug_config():
```

Comment:
not on this line in particular: the filename of the file this modifies has a typo: "custmize" should be "customize".  It should be renamed.

Reply:
ok, I will correct the name.

Reply:
yes, I am working resubmitting this PR based on the latest code.